src/hb-ot-font.cc

```diff
@@ -210,7 +210,11 @@ hb_ot_get_glyph_name (hb_font_t *font HB_UNUSED,
 		      void *user_data HB_UNUSED)
 {
   const hb_ot_face_t *ot_face = (const hb_ot_face_t *) font_data;
-  return ot_face->post->get_glyph_name (glyph, name, size);
+  hb_bool_t ret = ot_face->post->get_glyph_name (glyph, name, size);
+#ifndef HB_NO_OT_FONT_CFF
+  if (!ret) ret = ot_face->cff1->get_glyph_name (glyph, name, size);
+#endif
+  return ret;
```

Comment:
This, and next block, maybe better to rewrite as bunch of `if (...get_glyph_name (...)) return true;`.  Looks good to me otherwise.  Feel free to merge.

Reply:
made the suggested change

Reply:
Thanks.  There's a similar block in the other function (`get_glyph_from_name`) as well.

Reply:
ok... fixed `hb_ot_get_glyph_from_name` and `hb_ot_get_glyph_extents` as well taking this opportunity

Reply:
Thank you!!!!!